### PR TITLE
docs(relnote): FF122 - Support enabled for motion path features

### PR DESCRIPTION
**Pull request by dipikabh**
**Target release**: Firefox 122
**Release date**: Jan 23, 2024

----

### Description

The following preferences have been enabled by default in Firefox 122:

- [ray()](https://developer.mozilla.org/en-US/docs/Web/CSS/ray): `layout.css.motion-path-ray.enabled`
- [offset-position](https://developer.mozilla.org/en-US/docs/Web/CSS/offset-position): `layout.css.motion-path-offset-position.enabled`
- [offset-path](https://developer.mozilla.org/en-US/docs/Web/CSS/offset-path):
   - `layout.css.motion-path-basic-shapes.enabled`
   - `layout.css.motion-path-coord-box.enabled`
   - `layout.css.motion-path-url.enabled`

### Changes

- Experimental features: Removed entries for `ray()`, `offset-position`, and `url()`
- 122: Added notes for default availability of the above features
- ray: Removed experimental banner. Added "Formal syntax" (latest version is available in webref now).

### Related issues and pull requests

- Doc issue: https://github.com/mdn/content/issues/31099
- BCD PR: https://github.com/mdn/browser-compat-data/pull/21844
- Firefox bugs:
https://bugzilla.mozilla.org/show_bug.cgi?id=1598151
https://bugzilla.mozilla.org/show_bug.cgi?id=1598152
https://bugzilla.mozilla.org/show_bug.cgi?id=1598159

### CODE REVIEW

File: files/en-us/web/css/ray/index.md [hunk 1]
```diff
@@ -2,20 +2,13 @@
...
-{{CSSRef}}{{SeeCompatTable}}
+{{CSSRef}}
 
 The **`ray()`** [CSS](/en-US/docs/Web/CSS) [function](/en-US/docs/Web/CSS/CSS_Functions) defines the [`offset-path`](/en-US/docs/Web/CSS/offset-path) line segment that an animated element can follow. The line segment is referred to as "ray". The ray begins from an {{cssxref("offset-position")}} and extends in the direction of the specified angle. The length of a ray can be constrained by specifying a size and using the `contain` keyword.
 
-```css
-ray() = ray( <angle> && <size>? && contain? && [at <position>]? )
-<size> = closest-side | closest-corner | farthest-side | farthest-corner | sides
-```
-
```
Comment (dipikabh) [inline files/en-us/web/css/ray/index.md:18]: Removed this manually added syntax. "Formal syntax" shows the latest syntax now.

**Review by pepelsbey — APPROVED** [review]
LGTM!

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Comment by dipikabh** [comment]
Thanks for reviewing, @pepelsbey and @estelle!

**Pull request merged**